aggregators/1.0/modules/ROOT/pages/aggregators-time-example.adoc

```diff
@@ -0,0 +1,137 @@
+= Time-Based Aggregator Example - Mule 4
+
```

Comment:
Edit instructions: This is pretty similar to the previous PR you review yesterday, so you can just focus on the intro paragraph and the last paragraph after the Console logger message.